docs/src/user-manual/components.md

```diff
@@ -1,30 +1,165 @@
+in the Enduro Ingest user interface when relevant.
+
+Enduro's user interface is built using [Vue.js](https://vuejs.org/) and
+[Bootstrap](https://getbootstrap.com/)
+
+## Preservation engine
+
+This component manages the transformation of Processing Information Packages
+(PIPs) into Archival Information Packages (AIPs) in a preservation workflow
+during processing. The preservation engine is equipped with a number of tools
+and microservices to perform preservation tasks (such as characterization,
+metadata capture, and normalization, etc.), and communicates the outcome of
+these actions back to Enduro for display in the
+[user interface](#user-interface).
+
+Enduro Ingest currently supports two different preservation engines:
+
+* [Archivematica](https://archivematica.org) is an open-source digital
+  preservation system that is designed to maintain standards-based, long-term
+  access to collections of digital objects. It provides an integrated suite of
+  free and open-source tools that allows users to process digital objects and
+  generate Archival Information Packages (AIPs) in compliance with the Open
+  Archival Information System
+  ([OAIS](https://en.wikipedia.org/wiki/Open_Archival_Information_System))
+  functional model and other digital preservation standards and best practices,
+  define preservation policies, manage processing configurations, and more.
+  Archivematica includes a web-based dashboard for user interaction, as well as
+  a set of [API](https://en.wikipedia.org/wiki/API) endpoints that Enduro uses
+  to integrate with Archivematica and display preservation task outcomes in the
+  Enduro user interface.
+* [a3m](https://github.com/artefactual-labs/a3m) is a streamlined version of
+  Archivematica that is wholly focused on AIP creation. It does not have
+  external dependencies, integration with access systems, search capabilities,
+  or a graphical interface. It was designed to reduce the bulk of
+  Archivematica's extraneous functions for users operating at a large scale who
+  are more focused on throughput of digital objects for preservation.
+
+## Preservation storage service
+
+This component acts as an interface between the
+[preservation engine](#preservation-engine) and whatever storage has been
+configured for the long-term preservation of AIPs produced by Enduro workflows.
+The storage service manages the storage, movement, replication, and deletion of
+packages, and other related functions such as location management, encryption,
+fixity checking, and more.
+
+Currently Enduro uses the [Archivematica](https://archivematica.org) Storage
+Service ([AMSS](https://github.com/artefactual/archivematica-storage-service))
+when using Archivematica as the system's preservation engine. When using a3m,
+Enduro uses its own lightweight storage functionality to cover basic storage
+servicre requirements.
+
+-----
+
+* [Back to top](#components)
```

Comment:
I think you can cut these links too — the user can navigate to other pages or the top of this page with auto-generated TOC.

Reply:
Sure